languages.toml

```diff
@@ -4887,3 +4887,22 @@ indent = { tab-width = 2, unit = "  " }
 [[grammar]]
 name = "nearley"
 source = { git = "https://github.com/mi2ebi/tree-sitter-nearley", rev = "12d01113e194c8e83f6341aab8c2a5f21db9cac9" }
+
+[[language]]
+name = "kcl"
+scope = "source.kcl"
+injection-regex = "kcl"
+file-types = ["kcl"]
+comment-tokens = "//"
+indent = { tab-width = 2, unit = "  " }
+formatter = { command = "zoo" , args = ["kcl", "fmt", "-"] }
+language-servers = [ "kcl-lsp" ]
+block-comment-tokens = { start = "/*", end = "*/"}
+
+[language-server.kcl-lsp]
+command = "kcl-language-server"
+args = ["server", "--stdio"]
+
+[[grammer]]
+name = "kcl"
+source = { git = "https://github.com/KittyCAD/tree-sitter-kcl", rev = "1f9e40020b1bb76b31a7144b994bec1a559dc735"}
```

Comment:
It looks like this grammar repository doesn't have a license - could you suggest adding one upstream and update the `rev` here once one is included? Grammar repositories need to have licenses for the sake of packaging them

Reply:
Note, they do specify a license in their `Cargo.toml`, `pyproject.toml`, and `tree-sitter.json`, only a copy isn't included. PR open to add the copy https://github.com/KittyCAD/tree-sitter-kcl/pull/18